src/utils/fishData.js

```diff
@@ -1487,8 +1523,18 @@ const fishComponents = {
       ]
     },
     pink: {
-      rgb: [238, 142, 180],
-      knnData: [238, 142, 180],
+      rgb: [255, 153, 220],
+      knnData: [255, 153, 220],
+      exclusions: [
+        AppMode.FishVTrash,
+        AppMode.CreaturesVTrashDemo,
+        AppMode.CreaturesVTrash,
+        AppMode.FishShort
+      ]
+    },
+    darkPink: {
+      rgb: [255, 109, 172],
+      knnData: [255, 109, 172],
```

Comment:
Is it worth just unifying the `rgb` and `knnData` fields for colours in this file?

Reply:
yeah, probably. i thought it might come in handy for them to be separate, but it hasn't so far 🤷‍♀

Reply:
ah, nevermind. it looks like we rely on everything in `fishData` to define its `knnData`, so i'll leave that

Reply:
I don't think it should be difficult to change (eg here: https://github.com/code-dot-org/ml-activities/blob/babf882f895cb2e395790317c53d04c09191825c/src/demo/OceanObject.js#L237 though there will be tests to change too). I think it could be worth doing at some point but probably not as part of this PR.

Reply:
yeah, i changed it there, but we more-so rely on it in `initFishData`: 
https://github.com/code-dot-org/ml-activities/blob/babf882f895cb2e395790317c53d04c09191825c/src/utils/fishData.js#L1516-L1517

Reply:
Ah missed that spot